cmd/cloner/cloner.go

```diff
@@ -58,19 +59,22 @@ func main() {
 		log.Fatalf("wrong number of packages: %d", len(pkgs))
 	}
 	pkg := pkgs[0]
-	buf := new(bytes.Buffer)
-	imports := make(map[string]struct{})
+	cloneBuf := new(bytes.Buffer)
+	viewBuf := new(bytes.Buffer)
+	cloneImports := make(map[string]struct{})
+	viewImports := make(map[string]struct{})
```

Comment:
high level question: why merge this functionality into the cmd/cloner tool? It seems unrelated, or unrelated enough to be its own specific tool.

If they need to share some code, make a package?